docs/developer/payments/refunds.mdx

```diff
@@ -291,3 +303,39 @@ Below is an example order with multiple transactions and `totalRemainingGrant` a
   It also means that `totalChargedAmount` includes `amountChargePending`, and `totalRefundedAmount` includes `amountRefundPending`.
 - Authorized amounts, including `amountAuthorized` and `amountAuthorizePending`, are also considered processed. These amounts will contribute to the overall processed totals and impact the `overchargedAmount`.
   :::
+
+
+
+
+## Refund settings
+
+:::note
+Available from 3.22
+:::
+
+While `reason` field is optional and remains a plain text field, the `reasonReference` can be configured:
```

Comment:
IMO we should firstly add short explanation what is this refund setting, and why it's usefull.